commands/install.bee.inc

```diff
@@ -13,9 +13,21 @@ function install_bee_command() {
       'description' => bt('Install Backdrop and setup a new site.'),
       'callback' => 'install_bee_callback',
       'options' => array(
-        'db' => array(
-          'description' => bt("A database connection string of the form: 'mysql://user:pass@localhost/database_name'. See https://api.backdropcms.org/database-configuration for more details."),
-          'value' => bt('String'),
+        'db-name' => array(
+          'description' => bt("The database name."),
+          'value' => bt('Database name'),
+        ),
+        'db-user' => array(
+          'description' => bt("The username for the database."),
+          'value' => bt('Database username'),
+        ),
+        'db-pass' => array(
+          'description' => bt("The password for the database. "),
+          'value' => bt('Database password'),
+        ),
+        'db-host' => array(
+          'description' => bt("The host for the database relative to your webserver. Defaults to 'localhost'"),
```

Comment:
I think we should mention the fact that specifying a DB name but not host causes this to default. Also not sure what "relative to your webserver" means ('relative' makes it  sound like a path)...

```suggestion
          'description' => bt("The host for the database. Defaults to 'localhost' when a database name is provided."),
```

Reply:
Done.